src/io/accept.rs

```diff
@@ -17,21 +18,23 @@ impl Op<Accept> {
             unsafe { std::mem::zeroed() },
             std::mem::size_of::<libc::sockaddr_storage>() as libc::socklen_t,
         ));
-        Op::submit_with(
-            Accept {
-                fd: fd.clone(),
-                socketaddr,
-            },
-            |accept| {
-                opcode::Accept::new(
-                    types::Fd(accept.fd.raw_fd()),
-                    &mut accept.socketaddr.0 as *mut _ as *mut _,
-                    &mut accept.socketaddr.1,
-                )
-                .flags(libc::O_CLOEXEC)
-                .build()
-            },
-        )
+        CONTEXT.with(|x| {
```

Comment:
For many of these op constructors, I plan to have the weak handle in the socket object.